### DISCUSSION TITLE
Copilot extension does not activate in VSCode

### DISCUSSION THREAD

**Discussion by DemianParkhomenko** [community/community#52604] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Copilot extension does not work in VSCode, but other extensions work well. 

Firstly Copilot shows in 'Running Extensions': 
Command interactiveEditor.start appears multiple times in the commands section.

![image](https://user-images.githubusercontent.com/95881717/231404701-8c3978fd-683c-4688-a845-7a4b5346d029.png)

And after that, it shows the error and all VSCode freezes: 
  
![image](https://user-images.githubusercontent.com/95881717/231404938-7ed12af3-84b1-4fcf-afaf-521fa0dc49be.png)

The error persists: 
- with different versions of Github Copilot
- with only Github Copilot extension installed
- without any advanced settings for Copilot `github.copilot.advanced`

Operating System: Manjaro Linux 
KDE Plasma Version: 5.27.4
KDE Frameworks Version: 5.104.0
Qt Version: 5.15.8
Kernel Version: 6.1.23-1-MANJARO (64-bit)
Graphics Platform: X11

Copilot extension version: v1.82.15

**Comment by Akash1134** [comment]
Hi @DemianParkhomenko ,

We're sorry that you're facing issues using Copilot with VSCode.

We're seeing a few issues with VSCode on Windows specifically, and we're working on it.

In the meantime, a current workaround is to roll back to a previous version of Copilot.
<img width="648" alt="Screenshot 2023-04-12 at 5 03 23 PM" src="https://user-images.githubusercontent.com/95834130/231445410-2f3bbe98-b5d3-498e-81b6-e2f5b2c43a10.png">


Let me know how it goes.

Thanks!

**Reply by DemianParkhomenko** [reply, marked as answer]
Installed version 1.78. from VSCode and it works. 
![image](https://user-images.githubusercontent.com/95881717/231444384-c74e1331-00a8-4590-b79b-da69a914d4d0.png)

**Reply by Akash1134** [reply]
Thanks for the update @DemianParkhomenko Glad to know it helped. :)

**Reply by jakarina** [reply]
I am on Win11 and have this issue as well. As solution post suggested, downgrading to 1.78 works fine.

**Comment by pkws100** [comment]
I have the same problem ... all Versions > [1.78.9758] still not working

**Comment by stuartcallum** [comment]
I was able to solve this issue by going to version 1.79.10634
However, the issue reoccurs on the latest version (1.82.15)
![image](https://user-images.githubusercontent.com/92211506/233050860-411406c9-7b96-4794-8700-b780f77ee86f.png)

**Comment by DENNIS-CODES** [comment]
am a windows 11 user and the workaround is to down grade the copilot version to 1.78

**Reply by hosikiti** [reply]
Same on Mac OS 13.2 (22D49). All versions above 1.78 do not work in dev containers.

**Reply by jw834ksj43** [reply]
Also on windows 11. 1.78 fixed the issue as well.

**Comment by Jaska** [comment]
Same problem with Mac Os X 13.3.1 - Copilot doesn't start in dev containers. Installing version 1.78 works, but it's kind of annoying to keep doing that as vscode keeps updating the plugin.

[a comment by an automated account is not shown]
